run-ci.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+python="python3"
+if test -x "./.venv/bin/python3"; then
+  python="./.venv/bin/python3"
+fi
+
+FIRST_COMMIT_HASH=$( git log master..$( git branch --show-current ) --pretty=format:%H | tail -1 )
+LAST_COMMIT_HASH=$( git rev-parse HEAD )
+ARTICLES=$( git diff --diff-filter=d --name-only ${FIRST_COMMIT_HASH}^ ${LAST_COMMIT_HASH} "wiki/**/*.md" "news/*.md" )
```

Comment:
I've just noticed that it only takes committed changes into account due to `${FIRST_COMMIT_HASH}^` part. if I don't commit anything, the diff will be empty (and if I stage files, it will also be empty when requested without `--cached`). this may and will produce misleading results